lib/enum/arrays.rb

```diff
@@ -1,24 +1,36 @@
 class Array
+  attr_accessor :array
+
+  def initialize(array)
+    @array = array
+  end
 
   def sum_of_numeric_elements
+    array.select {|elem| elem.is_a?(Numeric)}.inject {|sum, elem| sum + elem}
```

Comment:
You can use symbol-to-proc to make your inject method shorter, like this: `.inject(:+)` Look up more on symbol-to-proc, it's pretty neat!